RoddyCore/src/de/dkfz/roddy/execution/io/SSHExecutionService.groovy

```diff
@@ -866,4 +871,43 @@ class SSHExecutionService extends RemoteExecutionService {
     @Override
     boolean canQueryFileAttributes() { return true; }
 
+    /**
+     * Stores a new job jobState info to an execution contexts job jobState log file.
+     *
+     * @param job
+     */
+    private static String jobStateInfoLine(String jobId, String code, String millis) {
+        return String.format("%s:%s:%s", jobId, code, millis)
+    }
+    private static String getJobStateInfoLineForJob(BEJob job) {
```

Comment:
Why is this method necessary? It looks partly like the method in LocalExecutionService? Maybe you can clarify it?